src/smc-webapp/editor.coffee

```diff
@@ -342,10 +343,7 @@ PUBLIC_ACCESS_UNSUPPORTED = ['terminal','latex','history','tasks','course', 'cha
 exports.public_access_supported = (filename) ->
     ext = filename_extension_notilde(filename)
     x = file_associations[ext]
-    if x?.editor in PUBLIC_ACCESS_UNSUPPORTED
-        return false
-    else
-        return true
+    return x?.editor in PUBLIC_ACCESS_UNSUPPORTED
```

Comment:
For what it is worth, I think PUBLIC_ACCESS_UNSUPPORTED is no longer used or valid -- what is supported is determined by which editors are *registered* to support public access.  See the directory `public/`.